app/views/microposts/_micropost.html.erb

```diff
@@ -1,10 +1,10 @@
 <tr>
-            <td>
+            <td id="blog-edit">
```

Comment:
@HrisikeshSinghaChowdhury single id can not be present multiple times in a single page. this partial is being called for every micropost in a loop. you need to set specific id (eg: "blog_<%= micropost.id %>") to that row and also not in <td> it should be is <tr> itself and full tr should be replaced in update.js.erb while updating the micropost

Reply:
ok understood